src/ConnectorRuntime.ts

```diff
@@ -47,6 +77,7 @@ export class ConnectorRuntime extends Runtime<ConnectorRuntimeConfig> {
             throw new Error(errorMessage);
         }
 
+        this.addMandatoryModulesConfiguration(connectorConfig);
```

Comment:
Wie bereits telefonisch besprochen: falls nicht zu viel Aufwand, dann vielleicht lieber die Pflichtmodule in die Default-Config packen, und dann über das JSON-Schema validieren, dass es enabled ist.

Reply:
bin jetzt einen zwischenweg gegangen, das aktuelle JSON-Schema validiert einfach nur die Module und es wäre zu kompliziert jetzt zusätzlich einzelne Module zu validieren